src/NuGet.Core/NuGet.Commands/RestoreSummary.cs

```diff
@@ -2,26 +2,150 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
+using NuGet.Configuration;
+using NuGet.Logging;
+using NuGet.Protocol.Core.Types;
 
 namespace NuGet.Commands
 {
     public class RestoreSummary
     {
+        public bool Success { get; }
+
         public string InputPath { get; }
 
-        public bool Success { get; }
+        public IEnumerable<string> ConfigFiles { get; }
```

Comment:
You may want to just keep these as lists and enumerate them here. I can't fully tell if it will end up evaluating these over and over or if it will be more efficient this way.